CHANGELOG.md

```diff
@@ -48,6 +48,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `go.opentelemetry.io/contrib/instrumentation/github.com/astaxie/beego/otelbeego` module is deprecated. (#4092, #4104)
 - The `go.opentelemetry.io/contrib/instrumentation/github.com/go-kit/kit/otelkit` module is deprecated. (#4093, #4104)
 - The `go.opentelemetry.io/contrib/instrumentation/github.com/Shopify/sarama/otelsarama` module is deprecated. (#4099)
+- The `go.opentelemetry.io/contrib/instrumentation/github.com/bradfitz/gomemcache/memcache/otelmemcache` module is deprecated. (#4164)
+- The `go.opentelemetry.io/contrib/instrumentation/github.com/gocql/gocql/otelgocql` module is deprecated. (#4164)
+- The `go.opentelemetry.io/contrib/zpages` module is deprecated. (#4164)
```

Comment:
I think the plan was not to deprecate zpages

Reply:
Correct I have forgotten to remove this line from changelog.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4164/commits/3dee7b5dc783e659262cd1cc1e269a88f4236a2b